modules/hyprlock/hm.nix

```
@@ -0,0 +1,22 @@
+{ config, lib, ... }:
+
+with config.lib.stylix;
+{
+  options.stylix.targets.hyprlock.enable =
+    mkEnableTarget "Hyprlock" true;
+
+  config = lib.mkIf (config.stylix.enable && config.stylix.targets.hyprlock.enable) {
+    programs.hyprlock.settings = {
+      background = {
+        path = config.stylix.image;
+      };
```

Consider simplifying the attribute set declaration:

```suggestion
      background.path = config.stylix.image;
```